annotator/pom.xml

```diff
@@ -33,6 +33,29 @@
       <artifactId>commons-cli</artifactId>
       <version>1.3</version>
     </dependency>
+    <dependency>
+        <groupId>javax.xml.bind</groupId>
+         <artifactId>jaxb-api</artifactId>
+        <version>2.3.0</version>
+     </dependency>
+    <dependency>
+        <groupId>com.h2database</groupId>
+        <artifactId>h2</artifactId>
+        <version>1.3.156</version>
+    </dependency>
```

Comment:
needed some database, otherwise there'd be some errors about db configuration. I haven't really tested the database annotator. Not sure if we are still using that?

Reply:
i dont think we use an h2 database at all, this might be related to spring batch specifically. I think it creates one in memory to store job and execution context information?